agent/statemanager/state_manager.go

```diff
@@ -50,7 +50,8 @@ const (
 	//   d) Add 'ENIAttachments' struct
 	//   e) Deprecate 'SteadyStateDependencies' in favor of 'TransitionDependencySet'
 	// 7)
-	//	 a) Add 'MetadataUpdated' field to 'api.Container'
+	//   a) Add 'MetadataUpdated' field to 'api.Container'
+	//   b) Add 'DomainNameServers' and 'DomainNameSearchList' in `api.ENI`
```

Comment:
Are we sure that this is going together with metadata?

Reply:
We'll see :) If not, its a merge conflict that'll need resolution